### PR TITLE
Added MTO Console Documentation

### PR DESCRIPTION
**Pull request by bnallapeta**
Describes MTO Console with all its features and screenshots

### CODE REVIEW

**Comment by bnallapeta** [comment]
need to wait for https://github.com/stakater/vocabulary/pull/70 to be merged to fix the vale issues on this.

File: content/explanation/console.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+The Showback feature is an essential financial governance tool, providing detailed insights into the cost implications of resource usage by tenant or namespace or other filters. This facilitates a transparent cost management and internal chargeback or showback process, enabling informed decision-making regarding resource consumption and budgeting.
+
+![image](../images/showback.png)
+
+## User Roles and Permissions
+
+### Administrators `:`
+
+Administrators have overarching access to the console, including the ability to view all namespaces and tenants. They have exclusive access to the IntegrationConfig, allowing them to define and manage system-wide settings and integrations.
```
Comment (SheryarButt) [inline content/explanation/console.md:48]: They can't define anything on the console, it's view only :smiling_face_with_tear:

File: content/explanation/console.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+## Dashboard Overview
+
+The dashboard serves as a centralized monitoring hub, offering insights into the current state of tenants, namespaces, templates and quotas. It is designed to provide a quick summary/snapshot of MTO's status and facilitates easier interaction with various resources such as tenants, namespaces, templates, and quotas.
+
+![image](../images/dashboard.png)
+
+### Tenants
+
+Here, admins have a bird's-eye view of all tenants, with the ability to delve into each one for detailed examination and management. This section is pivotal for observing the distribution and organization of tenants within the system. More information on each tenant can be accessed by clicking the view option against each tenant name.
```
[the remark this answers is not in the document]
Comment (bnallapeta) [reply content/explanation/console.md:15]: https://github.com/stakater/mto-docs/blob/console/content/explanation/console.md#live-yaml-configuration-and-graph-view

added that exclusively here.

File: content/explanation/console.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+# MTO Console
+
+## Introduction
+
+The Multi Tenant Operator (MTO) Console is a comprehensive user interface designed for both administrators and tenant users to manage multi-tenant environments. The MTO Console simplifies the complexity involved in handling various aspects of tenants and their related resources.
+
+## Dashboard Overview
+
+The dashboard serves as a centralized monitoring hub, offering insights into the current state of tenants, namespaces, templates and quotas. It is designed to provide a quick summary/snapshot of MTO's status and facilitates easier interaction with various resources such as tenants, namespaces, templates, and quotas.
```
[the remark this answers is not in the document]
Comment (bnallapeta) [reply content/explanation/console.md:9]: Modified.

File: content/explanation/console.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
+# MTO Console
+
+## Introduction
+
+The Multi Tenant Operator (MTO) Console is a comprehensive user interface designed for both administrators and tenant users to manage multi-tenant environments. The MTO Console simplifies the complexity involved in handling various aspects of tenants and their related resources.
+
+## Dashboard Overview
+
+The dashboard serves as a centralized monitoring hub, offering insights into the current state of tenants, namespaces, and quotas. It is designed to provide a quick summary/snapshot of MTO resources' status. Additionally, it includes a Showback graph that presents a quick glance of the seven-day cost trends associated with the namespaces.
```
Comment (SheryarButt) [inline content/explanation/console.md:9]: seven-day cost trends associated with the namespaces -> seven-day cost trends associated with the namespaces/tenants based on the logged-in user

**Review by SheryarButt — APPROVED** [review]

**Pull request merged**